src/shared/components/HomeAddButton/HomeAddButton.tsx

```diff
@@ -0,0 +1,21 @@
+import * as styles from './HomeAddButton.css';
+import ArrowRight from "@assets/icon/ic_arrow_right_gray70_16.svg";
+
+type ViewMoreProps = {
+  keyword: string; // 뮤지컬 랭킹, 할인티켓, 오픈 예정 공연
+};
```

Comment:
keyword값이 정해져있다면 

```suggestion
type KeywordType = '뮤지컬 랭킹' | '할인티켓' | '오픈 예정 공연';

interface ViewMoreProps {
  keyword: KeywordType;
}
```
이런식으로 유니온 타입으로 작성해주면 휴먼에러를 방지할 수 있습니닷

Reply:
우와 감사합니다